external/wakaama/core/er-coap-13/er-coap-13.c

```diff
@@ -45,7 +45,7 @@
 
 #include "er-coap-13.h"
 
-#include "liblwm2m.h"			/* for lwm2m_malloc() and lwm2m_free() */
+#include <protocols/liblwm2m.h>			/* for lwm2m_malloc() and lwm2m_free() */
```

Comment:
The Wakaama is in external folder. Do you want to modify this in all of wakaama files?
How about add including path of "external/include/protocols" in makefile?
That is easier to maintain I think.

Reply:
unfortunately wakaama also refers er-coap in "wakaama/core"  if I didn't change an include path. I should remove or sync the file liblwm2m.h in "wakaama/core". Actually I don't know which is better, but I think chaning path is clear than other methods.